### ISSUE TITLE
[Bug]: internal link not copied in clipboard when a document is being edited in NC 25.0.3

### ISSUE THREAD

**Issue by dorianne-arawa** [nextcloud/server#36606]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

When I open a document (editing mode) on Nextcloud 25.0.3 and copy the internal link into my clipboard, the link is not really copied.

### Steps to reproduce

1. Go to Nextcloud Files
2. Open and edit a document (.md or .odt for ex), on Nextcloud Office for example (collabora online)
3. Open the right panel and go in the "Share" tab
4. Click on the "Internal link" button
5. The link is noted as "copied on clipboard"
6. Past this internal link into a browser, or notpad, or whatever
7. The paste doesn't display the internal link (only what I copied into my clipboard before the internal link, or nothing if my clipboard is empty)

### Expected behavior

When I paste the internal link copied to my clipboard, I would like this link to be written/displayed.

For the moment, when I paste this internal link (in my browser, or just on a note pad on my computer) nothing happens (or the last words in my clipboard, before the copy of the internal link, appear)

### Installation method

None

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 7.4

### Web server

None

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [X] LDAP/ Active Directory
- [X] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
i can't execute this command line on my device and can't have access to the nextcloud server by command line.
```

**Comment by dorianne-arawa** [comment]
Seems linked to this issue, but with the difference that it happens already on NC 25.0.3 and not only on NC 26.0.0.4
https://github.com/nextcloud/server/issues/36582

**Comment by szaimen** [comment]
Should be fixed with 25.0.4

**Comment by dorianne-arawa** [comment]
Perfect, thank you @szaimen !
